### PR TITLE
#9381: fix missing sidepanel tab title

### LINKED ISSUE #9381: Page Editor: Missing sidepanel tab title
## Describe the bug

When I have a sidepanel mod component, the sidebar is open, and I select the mod containing that sidepanel mod component or another mod component in that mod, then I do not see the tab title in the sidebar

## To Reproduce

1. Create or install a mod with a side panel mod component
2. Open the sidebar
3. Open the Page Editor
4. Select the mod with the side panel mod component (but do not select the sidepanel mod component)

## Actual behavior

The tab title is not visible

## Expected behavior

The tab title is visible

## Screenshots/Loom

https://pixiebrix.slack.com/archives/C023KL47XV4/p1730223727362799

## Desktop (please complete the following information):

- Extension Version: 1.2.7-alpha.0 (main)

### PR DESCRIPTION
**Pull request by twschiller**
## What does this PR do?

- Fixes #9381

## Discussion

- ~@grahamlangford I was not able to reproduce issue #9381. Let's see if it's reproduced on CI~

For more information on our expectations for the PR process, see the
[code review principles doc](https://www.notion.so/pixiebrix/Code-Review-Principles-1ce7276b82a84d2a995d55ad85e1310d?pvs=4)

### CODE REVIEW

**Comment by grahamlangford** [comment]
I'm not sure what's going on. I've been able to consistently reproduce with fresh installs across multiple browsers

**Comment by twschiller** [comment]
> I'm not sure what's going on. I've been able to consistently reproduce with fresh installs across multiple browsers

@grahamlangford does the test in this PR pass locally for you? (As an E2E test and/or manually?)

File: src/pageEditor/toolbar/ReloadToolbar.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ const ReloadToolbar: React.FunctionComponent<{
       }),
       { isSelectedInEditor: true, runReason: RunReason.PAGE_EDITOR_RUN },
     );
-  }, [asDraftModComponent, modComponentFormState]);
+  }, [asDraftModComponent, getOptionsArgsForModId, modComponentFormState]);
```
Comment (twschiller) [inline src/pageEditor/toolbar/ReloadToolbar.tsx:121]: Not sure what's going on here? Compiling fine for me locally and I can see the variable in the GitHub diff

image

Comment (grahamlangford) [reply src/pageEditor/toolbar/ReloadToolbar.tsx:120]: Pull? Looks like it requires the whole draft mod state, which includes variables

**Comment by twschiller** [comment]
@grahamlangford LGTM

**Review by grahamlangford — APPROVED** [review]

**Pull request merged**